walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,148 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeIsIgnoredInResult() {
+    // Have each person have different events. The optional 
+    // attendee should be ignored because there is no time
+    // that accommodates the optional attendee.
+    //
+    // Events  : |^^^^^|--A--|^^C^^|--B--|^^^^^|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
```

Comment:
BTW, in order to comply with the figure above. You can call the Events: evantA, eventB and eventC.

Reply:
Ack - I think I am trying to show the event labelled with who is attending rather than the event name.